core/node/events/parsed_event.go

```diff
@@ -16,6 +16,7 @@ import (
 type ParsedEvent struct {
 	Event         *StreamEvent
 	Envelope      *Envelope
+	Tags          *Tags
```

Comment:
This seems a bit redundant: this field is readily available as `Event.Tags`, if brevity is desired, it's possible to add `Tags()` method to `ParsedEvent`